### DISCUSSION TITLE
[Pixi-Spine] Mask not working correctly; works fine in spine-ts player

### DISCUSSION THREAD

**Discussion by randrade23** [pixijs/pixijs#8076] [Q&A]
### Expected Behavior

![image](https://user-images.githubusercontent.com/2223723/147558733-7dce4ba1-c635-4530-ac25-36b701e41dba.png)

### Current Behavior

![image](https://user-images.githubusercontent.com/2223723/147561753-3520abe9-6cca-46c9-8211-89655c1b4591.png)

### Steps to Reproduce

Load the spine in the demo page and start the "SUN/PIECES/CENTER" animation.

### Environment

- **`pixi.js` version**: _6.2.1 / WebGL 2_
- **Browser & Version**: _Chrome 96.0.4664.110_
- **OS & Version**: _Windows 11_
- **Running Example**: [Demo](https://d5lfyopf9muc4.cloudfront.net/spine-demo/index.html)

Hi,

I have a Spine animation that renders correctly in the Spine editor and in the spine-ts player, but when using pixi-spine there is a mask that does not apply correctly - it does not clip some deformed bones.

I had already opened an issue on the pixi-spine repository but @ivanpopelyshev suggested I opened a thread in the discussions part of the Pixi repo.

**Comment by ivanpopelyshev** [comment]
This is most probably a mask issue
